protocols/grpc/utils.go

```diff
@@ -15,6 +21,12 @@ import (
 	"github.com/opencontainers/runtime-spec/specs-go"
 )
 
+// OCI config file
+const (
+	OCIConfigFile     string      = "config.json"
```

Comment:
This looks a bit odd. For consistency, please use the same capitalisation - I'd suggest changing this to use a `oci` prefix to keep it private.

Reply:
This was done access it from the tests, but moving the tests over to `utils_tests.go` is probably better right?

Reply:
Yes, since you are testing a function under utils.go, the test needs to go under utils_test.go.